### ISSUE TITLE
update image links in readme

### ISSUE THREAD

**Issue by actionless** [gusbemacbe/suru-plus#130]
hi gus!
just accidentally noticed this page have broken image links: https://github.com/gusbemacbe/suru-plus/blob/master/tutorials.md

probably as you moved screenshots from ./beta/ dir to ./images/

**Comment by gusbemacbe** [comment]
Hi @actionless !

Thank you for reporting, I will do it when I will release the new version of Suru++. Once I will release Suru++ family, I will upgrade your repository. 

At the comment I am solving the bugs of GNOME++ and of Java-written cryptography project.

**Comment by gusbemacbe** [comment]
I will update the next week, @actionless. 

A Brazilian guy in the Brazilian forum is asking if Oomox has Mint-Y theme or will support it.

**Comment by gusbemacbe** [comment]
Will it be possible to add inspecting elements feature to Oomox GTK and to Oomox, @actionless ?

**Comment by actionless** [comment]
> A Brazilian guy in the Brazilian forum is asking if Oomox has Mint-Y theme or will support it.
yes, here short instructions/example on how to add new theme: https://github.com/themix-project/oomox/issues/167

> Will it be possible to add inspecting elements feature to Oomox GTK and to Oomox,
sorry, i can't understand the question

**Comment by gusbemacbe** [comment]
Hello @actionless, I have just indicated him it, but he wants to use the default themes Mint-Y and Mint-Y-Dark-Grey to modify.

He wants to inspect elements (calibrate) any object or any colour of GTK body like you do it with Google Chrome and Firefox. For example; `GTK_DEBUG=interactive nemo`. 

He is asking if you offer inspecting and modify this part like:

![image](https://user-images.githubusercontent.com/2257024/59592356-43630d00-90c6-11e9-8b38-b95111744a7a.png)

He indicated the video https://www.youtube.com/watch?v=JC-QjEiDjEU, in the which he would like to modify live the GTK SASS file, while saving, the window auto-syncs.

**Comment by actionless** [comment]
> For example; GTK_DEBUG=interactive nemo

yeah, that's it, although i usually press Ctrl+Shift+D while some app is already running

**Comment by gusbemacbe** [comment]
@actionless , with which app?

**Comment by gusbemacbe** [comment]
@actionless , with which app (which the app changes itself live)?

**Comment by actionless** [comment]
in any app, just edit the code in "CSS" tab of the inspector, not in the theme sources

**Comment by gusbemacbe** [comment]
Hi @actionless !

He tried to do what you did, but while he changed the colour in CSS while the window did not change colour live.

Ah, I saw that it is SASS Watcher.
